### PR TITLE
Taboola Bidder Adapter:  Change Endpoint 

### PR DESCRIPTION
**Pull request by ahmadlob**
changes in the server endpoint, using query params for publisher id and exchange params instead of path params

### CODE REVIEW

File: static/bidder-info/taboola.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/PS/auction/{{.GvlID}}/{{.PublisherID}}"
+endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/PS/auction?exchange={{.GvlID}}&publisher={{.PublisherID}}"
```
Comment (gargcreation1992) [inline static/bidder-info/taboola.yaml:1]: Based on your description in the PR that this addresses the issue #2612, this endpoint is still using the dynamic host. You would need to use the static domain. For example, in this case, `MediaType` macro needs to be moved out as query param.

Comment (gargcreation1992) [reply static/bidder-info/taboola.yaml:1]: @ahmadlob This is a fine change to make but this change definitely not addressing the issue #2612. For static host, macro should not be included in the endpoint domain.
The above host is still dynamic.
Please change the description of the PR to reflect the changes that you are doing.

Comment (ahmadlob) [reply static/bidder-info/taboola.yaml:1]: @gargcreation1992 changed the PR description and title

**Review by gargcreation1992 — APPROVED** [review]
Thank you for changing the description. Approving it now!
Please be mindful that this issue does not address #2612

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**